Source/WebKit/NetworkProcess/webrtc/NetworkRTCMonitor.cpp

```diff
@@ -215,7 +215,8 @@ static HashMap<String, RTCNetwork> gatherNetworkMap()
 
         networkMap.ensure(networkKey, [&] {
             std::span<const char> name { iterator->ifa_name, strlen(iterator->ifa_name) };
-            std::span<const char> description { networkKey.utf8().data(), networkKey.utf8().length() };
+            auto descriptionString = networkKey.utf8();
+            std::span<const char> description { descriptionString.data(), descriptionString.length() };
             return RTCNetwork { { name }, { description }, address->second, prefixLength, interfaceAdapterType(name), 0, 0, true, false, scopeID, { } };
```

Comment:
... and pass `description.span()` here.

Reply:
Right, I'll use span here and for name as well.